packages/driver/src/cy/commands/actions/click.ts

```diff
@@ -117,11 +117,67 @@ export default (Commands, Cypress, cy: $Cy, state, config) => {
         // figure out the options which actually change the behavior of clicks
         deltaOptions = $utils.filterOutOptions(options, defaultOptions)
 
-        options._log = Cypress.log({
-          message: deltaOptions,
-          $el,
-          timeout: options.timeout,
-        })
+        // create a new object with all user-provided options and exclude null values
+        const clickOptions = Object.keys(userOptions)
```

Comment:
@Manuel-Suarez-Abascal I was testing this a bit locally, and this test case results in:

```
.click(5, 10, { position: 'topRight' })
```

```
- click topRight, 5, 10
```
which is unexpected.

Any reason we could simplify this down to an array of the passed args & figure out undefined values?
```
const message = [positionOrX, y, userOptions].filter((e) => !!e)
```

Reply:
@Manuel-Suarez-Abascal Do you think you'll have time to address this feedback? I tend to agree with Emily that this could probably be simplified to utilizing the `userOptions` and filtering out undefined values. Could also utilize the [stringify util](https://github.com/cypress-io/cypress/blob/236b1816667720e1f0295699a148b84c25d7c04f/packages/driver/src/cypress/utils.ts#L223) instead of manually doing it.

Reply:
@chrisbreiding @emilyrohrbough I haven't had the time to address these comments. Will try to find to do it this week, but feel to push the code to the branch if you have the capacity. I'll ping you when it's done :)

Reply:
> @Manuel-Suarez-Abascal I was testing this a bit locally, and this test case results in:
> 
> ```js
> .click(5, 10, { position: 'topRight' })
> ```
> 
> ```
> - click topRight, 5, 10
> ```
> 
> which is unexpected. worth adding this test case here.
> 
> Any reason we couldn't simplify this down to an array of the passed args & figure out undefined values?
> 
> ```js
> const message = [positionOrX, y, userOptions].filter((e) => !!e)
> ```

I tested on my side & you're 100% correct...before doing the necessary changes in the code to fix this unexpected behavior. What would the expected output of `.click(5, 10, { position: 'topRight' })`

Reply:
> What would the expected output of .click(5, 10, { position: 'topRight' }) ?

`position` isn't a valid option and is ignored by the implementation, so I would say it should output `-click 5, 10` since those are the coordinates where the click is fired.